deblur/test/test_workflow.py

```diff
@@ -91,8 +90,11 @@ def test_sequence_generator_fastq(self):
         self.assertTrue(obs[0][1].startswith(first_few_nucs))
 
     def test_sequence_generator_invalid_format(self):
-        with self.assertRaises(skbio.io.FormatIdentificationWarning):
-            next(sequence_generator(join(self.test_data_dir, 'readme.txt')))
+        allres = []
+        with self.assertWarns(UserWarning):
```

Comment:
Can you add a warning message here?   It can be something like

```
msg = "input file readme.txt does not appear to be FASTA or FASTQ"
with self.assertWarns(UserWarning, msg):
...
```